### ISSUE TITLE
Make tests less boilerplate-y

### ISSUE THREAD

**Issue by wokalski** [dialohq/stork#32]
When I was reviewing #31 I noticed that the tests are extremely boilerplate-y without a need. They seem to be duplicated between tests, too. It'd be good to reduce this clutter because otherwise it's super hard to make sense of them.

**Comment by tsnobip** [comment]
yeah, I agree, what do you have in mind to improve the situation?

**Comment by wokalski** [comment]
I had the simplest idea to just make the atd files tiny. Like one line definitions.
